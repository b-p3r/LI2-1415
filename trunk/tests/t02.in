3 3
DAB
MOS
RTB
