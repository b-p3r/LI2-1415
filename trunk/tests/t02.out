A
ADO
AMO
AMOR
AMOS
AO
AOS
AS
ASO
ASTRO
BASTO
BOA
BOAS
BOBA
BODA
BODAS
BOM
DA
DAMOS
DAO
DAS
DO
DOA
DOAM
DOAS
DOBA
DOM
DOR
DOS
DOSA
MA
MAO
MAOS
MAS
MASTRO
MO
MOA
MOAS
MODA
MODAS
MOR
MOS
O
OS
RO
ROA
ROAM
RODA
RODAM
RODAS
ROMA
ROMAS
ROSA
SA
SABOR
SADO
SAM
SAO
SO
SOA
SOAM
SOB
SOBA
SODA
SOM
SOMA
SOR
TO
TOA
TODA
TODAS
TOM
TOMA
TOMAS
